java/test/src/main/java/io/ray/test/RuntimeEnvTest.java

```diff
@@ -0,0 +1,66 @@
+package io.ray.test;
+
+import io.ray.api.ActorHandle;
+import io.ray.api.Ray;
+import io.ray.api.runtimeenv.RuntimeEnv;
+import io.ray.api.runtimeenv.RuntimeEnvBuilder;
+import io.ray.runtime.util.SystemUtil;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(groups = "cluster")
+public class RuntimeEnvTest extends BaseTest {
+
+  @BeforeClass
+  public void setUp() {
+    /// This is used to test that actors with runtime envs will not reuse worker process.
+    System.setProperty("ray.job.num-java-workers-per-process", "2");
+  }
+
+  private static class A {
+
+    public String getEnv(String key) {
+      return System.getenv(key);
+    }
+
+    public int getPid() {
+      return SystemUtil.pid();
+    }
+  }
+
+  public void testEnvironmentVariable() {
+    int pid1 = 0;
+    int pid2 = 0;
+    {
+      RuntimeEnv runtimeEnv =
+          new RuntimeEnvBuilder()
```

Comment:
Do we need to put `RuntimeEnvBuilder ` into `RuntimeEnv `? like:
```
RuntimeEnv runtimeEnv =
          new RuntimeEnv.RuntimeEnvBuilder()
```
I'm not sure. This two ways are all good for me. @raulchen what do you think?